nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java

```diff
@@ -85,6 +95,12 @@ public void testFetchPackage() throws Exception
     assertSuccessResponseMatches(resp, AGRICOLAE_PKG_FILE_NAME_131_TGZ);
   }
 
+  @Test
+  public void testFetchNotFoundPackage() throws Exception
```

Comment:
testFetchNotExistingPackage ?

Reply:
Done in 47bd5fb9dc9524a372cb0a1cd8218dad8aa45f03